MinGW (like MSVC) has _aligned_malloc() and _aligned_free(), not
posix_memalign(): key fbgemmAlignedAlloc()/fbgemmAlignedFree() on _WIN32.

diff --git a/src/Utils.cc b/src/Utils.cc
index 184af11..772c633 100644
--- a/src/Utils.cc
+++ b/src/Utils.cc
@@ -369,7 +369,7 @@ void* fbgemmAlignedAlloc(
     size_t size,
     bool raiseException /*=false*/) {
   void* aligned_mem = nullptr;
-#ifdef _MSC_VER
+#ifdef _WIN32
   aligned_mem = _aligned_malloc(size, align);
 #else
   int ret = posix_memalign(&aligned_mem, align, size);
@@ -385,7 +385,7 @@ void* fbgemmAlignedAlloc(
 }
 
 void fbgemmAlignedFree(void* p) {
-#ifdef _MSC_VER
+#ifdef _WIN32
   _aligned_free(p);
 #else
   free(p);
